content/blog/meet-our-new-hpe-dev-team-representative-at-hpe-technology-and-solution-summit-2023.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Meet our new HPE DEV Team Representative  at HPE Technology and Solution
+  Summit 2023!
+date: 2023-03-30T13:29:10.157Z
+author: Frederic Passeron
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+tags:
+  - HPE TSS
+  - HPE DEV
+---
+![](/img/tss2023.png "HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023")
+
+HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023 (HPE TSS 2023) goes physical again this year in sunny Barcelona. After several virtual editions, you will have the pleasure to engage with HPE Developer Community representative. All our sessions will be hold by our new team member : Mathieu Losmede. 
+
+H﻿PE TSS is the largest HPE technical event taking place in EMEA. Whether you are an HPE Presales or Partner, you will have the opportunity to attend technical breakout, Hands-on Labs, as well as strategic keynotes, and also finally get to meet your peers in person!
+
+T﻿he final decision to turn the event into a physical one came late this year. As a result, the HPE DEV team will not be able to bring the full physical Hack Shack back in HPE TSS 2023. However, we still come with a few sessions, some Hands-on labs, and a few surprises...
+
+**Meet our new team member: Mathieu Losmede**
+
+M﻿athieu joined our team last summer. He came in with a strong HPE GreenLake background. He was part of the xxxx team where he worked on..... . As the team focus lately shifted to HPE GreenLake Cloud Platform, his arrival  was very fortunate. Mathieu will be all over the place in Barcelona. One might wonder whether he managed to discover the secret of ubiquity...He will delivering several technical breakouts either on his own or in partnership with some other Business Unit representatives (HPE GreenLake, Storage, Compute, and Aruba). He will also be co presenting in other BUs ' sessions. On top of these, he will deliver as well three Hands-on labs based on our well-known [Workshops-on-Demand](https://developer.hpe.com/hackshack/workshops) project.
```

Comment:
First sentence, expand a bit: "Mathieu joined the HPE Developer Community team last summer."
Third sentence obviously still needs to be filled in :-). 
Fourth sentence: "As our team's focus has recently shifted more heavily towards the HPE GreenLake Cloud Platform, we were very fortunate to have him join our group."

I would then add a space/carriage return so that the next paragraph starts with "Mathieu will be all over the place..."
Place a period after "ubiquity." (not the three dots/elipsis).

Take out the word "some" prior to "other Business Unit representatives", make the B and U lower case and add (BU) after it. So it would read: "... in partnership with other business unit (BU) representatives (HPE GreenLake..."

"co presenting" should be "co-presenting"

There's a space between the apostrophe and "BUs" that should be removed. (the apostrophe should be right after the "s")

I would change "On top of these" to "On top of all this"

Take out "as well", make "Hands-on" --> "hands-on"
So the last sentence would read "On top of all this, he will deliver three hands-on labs based on..."